Readly/Data/Models/Joke.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Joke.swift
+//  Readly
+//
+//  Created by Viorel Rinja on 4/24/21.
+//
+
+import Foundation
+import ObjectMapper
+
+class Joke: Mappable {
+    var id: Int?
+    var setup: String?
+    var punchline: String?
+    
+    required init?(map: Map) {}
+    
+    func mapping(map: Map) {
+        id <- map["id"]
+        setup <- map["setup"]
+        punchline <- map["punchline"]
+    }
+}
```

Comment:
@viorelrj There is more elegant way of performing data decoding.
It is built in mechanism in swift which is based on [Codable Protocol](https://developer.apple.com/documentation/swift/codable) also take a look here [Encoding and Decoding Custom Types](https://developer.apple.com/documentation/foundation/archives_and_serialization/encoding_and_decoding_custom_types).
Please refactor and remove `ObjectMapper` from project